source/diagnostics.tex

```diff
@@ -825,7 +825,7 @@
 exposes the \libheader{cerrno} macros is unspecified.
 
 \pnum
-The \tcode{is_error_code_enum} and \tcode{is_error_condition_enum} may be
+Templates \tcode{is_error_code_enum} and \tcode{is_error_condition_enum} may be
```

Comment:
The most straightforward fix is to assume a dropped word "templates" in the right spot:
```suggestion
The \tcode{is_error_code_enum} and \tcode{is_error_condition_enum} templates may be
```

Reply:
Fixed.